lib/semian/unprotected_resource.rb

```diff
@@ -36,5 +36,12 @@ def request_allowed?
 
     def mark_failed(_error)
     end
+
+    def mark_success
+    end
+
+    def circuit_breaker_shared?
+      false
+    end
```

Comment:
If... you're referring to `circuit_breaker_shared?`, then I'm going to remove it along with making `#shared?` private. If it was the `mark_success`, there was a test somewhere that checked the `#public_instance_methods` of both types of the `Resource` wrappers to make sure they were the same.